components/webui/webui_resources.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_WEBUI_WEBUI_RESOURCES_H_
+#define BRAVE_COMPONENTS_WEBUI_WEBUI_RESOURCES_H_
+
+#include <string_view>
+#include <vector>
+
+#include "base/containers/flat_map.h"
+#include "ui/base/webui/resource_path.h"
+#include "ui/base/webui/web_ui_util.h"
+
+namespace webui {
+
+struct LocalizedString;
+struct ResourcePath;
```

Comment:
I guess you don't need these forwards, because you added `#include` lines above.

Reply:
Fixed (removed the unnecessary includes and kept the forward declaration): https://github.com/brave/brave-core/commit/28844bde3166f309f9a3e7c9b72c59646c65d4fa